templates/desktop/developers.html

```diff
@@ -57,7 +57,7 @@
 
 <section class="row strip-light">
     <div class="strip-inner-wrapper">
-        <h2>What&rsquo;s new in {{lts_release_no_point}}?</h2>
+        <h2>What&rsquo;s new in Ubuntu {{lts_release_full}}?</h2>
         <p class="eight-col">Alongside support for the very latest hardware, Ubuntu {{lts_release_no_point}} includes new versions of many core apps and developer technologies.</p>
```

Comment:
Copy doc says "Ubuntu 16.04 is the latest LTS release. Alongside support for the very latest hardware, this release includes new versions of many core apps and developer technologies."